armi/materials/thoriumOxide.py

```diff
@@ -106,8 +107,6 @@ def meltingPoint(self):
 
 
 class ThO2(ThoriumOxide):
-    """
-    Another name for ThoriumOxide.
-    """
+    """Another name for ThoriumOxide"""
```

Comment:
Needs  period. (we need to make pydocstyle part of our auto-linting! http://www.pydocstyle.org/en/stable/)

Reply:
fixed